multiclusterhub/charts/search-prod/templates/aggregator-service-account.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: search-aggregator
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822